src/components/views/join/JoinEngage.tsx

```diff
@@ -0,0 +1,169 @@
+import React from "react";
+import styled from "@emotion/styled";
+import Image from "next/image";
+import { Giv_900, Giv_700 } from "../../styled-components/Colors";
+import { H2, Lead } from "../../styled-components/Typography";
+import Card from "../../GeneralCard";
+import config from "../../../../config";
+import YellowFlower from "../../../../public/images/yellow_flower.svg";
+import BlueFlower from "../../../../public/images/blue_flower.svg";
+import discord_icon from "../../../../public/images/discord_icon.svg";
+import discourse_icon from "../../../../public/images/discourse_icon.svg";
+import github_icon from "../../../../public/images/github_icon.svg";
+import telegram_icon from "../../../../public/images/telegram_icon.svg";
+import medium_icon from "../../../../public/images/medium_icon.svg";
+import twitter_icon from "../../../../public/images/twitter_icon.svg";
```

Comment:
Please run `yarn run lint:fix` before pushing commits. It will fixes all ESLint errors for you.
Like in giveth-next, we use single quote and no semicolon